commons/src/main/java/com/powsybl/openrao/commons/Unit.java

```diff
@@ -42,6 +42,15 @@ public String toString() {
         return symbol;
     }
 
+    public static Unit getEnum(String value) {
```

Comment:
Necessary ? Unit seems to be serialized properly so why handle case sensitivity when deserializing ?

Reply:
No reason, I am going to fix it.